src/id-data-controls.js

```diff
@@ -39,115 +39,117 @@ class IdDataControls extends preact.Component {
     render() {
         if (Privacy.isAllowed(PRIVACY_ACTIONS.SAVE_ID_DATA)) {
             return (
-                <div id="id-data-controls-container" className="narrow-page">
-                    <DynamicInputContainer
-                        key="id-data-controls"
-                        id="id-data-controls"
-                        onChange={this.handleCustomChange}
-                        fields={this.state.custom_id_data}
-                        title={t('saved-data', 'id-data-controls')}
-                        hasPrimary={false}>
-                        <IntlProvider scope="id-data-controls" definition={I18N_DEFINITION}>
-                            <div>
-                                <p>
-                                    <MarkupText id="saved-data-explanation" />
-                                </p>
-                                <div className="form-group">
-                                    <input
-                                        type="checkbox"
-                                        id="always-fill-in"
-                                        className="form-element"
-                                        checked={SavedIdData.shouldAlwaysFill()}
-                                        onChange={event => {
-                                            SavedIdData.setAlwaysFill(!SavedIdData.shouldAlwaysFill());
-                                        }}
-                                    />
-                                    <label htmlFor="always-fill-in">
-                                        <Text id="always-fill-in" />
-                                    </label>
-                                </div>
-
-                                <div className="form-group" style="width: 100%; display: table;">
-                                    <div className="col40">
-                                        <strong>
-                                            <label htmlFor="name-input-value-fixed-id-data">
-                                                {t('name', 'generator')}
-                                            </label>
-                                        </strong>
-                                    </div>
-                                    <div className="col60" style="padding-left: 10px; box-sizing: border-box;">
-                                        <InputControl
-                                            id="name-input"
-                                            suffix="fixed-id-data"
-                                            onChange={e => this.handleFixedChange('name', e)}
-                                            value={this.state.fixed_id_data['name']}
+                <main>
```

Comment:
How about adding the main tag just at the end here:

https://github.com/datenanfragen/website/blob/f07be204fdc5663934cdb32b19679f423b7cbbc9/src/id-data-controls.js#L241-L247

Reply:
That's a much better way of doing that, I've made the changes I think